cider/exec/module/batch/CiderBatchUtils.cpp

```diff
@@ -67,6 +67,11 @@ ArrowSchema* allocateArrowSchema() {
   return ptr;
 }
 
+ArrowSchema* allocateArrowSchema(const ArrowSchema& schema) {
+  ArrowSchema* ptr = new ArrowSchema(schema);
```

Comment:
use allocator in runtimemodule?

Reply:
There is no other place needs, seems unnecessary now.